javascript/src/UploadField.js

```diff
@@ -430,13 +430,14 @@ $.entwine('ss', function($) {
 			}
 
 			if (iframe.attr('src') == 'about:blank') {
+				var disabled = this.siblings();
```

Comment:
Ah yes, this was because of 'use strict'; the variables just have to be declared properly.